04_setup_ironic.sh

```diff
@@ -29,11 +29,12 @@ DOCKERFILE=$(mktemp --tmpdir "release-update--XXXXXXXXXX")
 _tmpfiles="$_tmpfiles $DOCKERFILE"
 echo "FROM $OPENSHIFT_RELEASE_IMAGE" > $DOCKERFILE
 
-# To build custom images is highly recommended to build a base image first.
-# Build a base image if we set a custom repo file in the config file and
-# the file exists
-if [[ -n ${CUSTOM_REPO_FILE:-} ]]; then
-    ./build-base-image.sh ${BASE_IMAGE_DIR:-base-image} $CUSTOM_REPO_FILE
+# Build a custom base image for the ironic images.
+# This may be necessary in case we want to rebuild the base image from
+# scratch or can't for some reason get the base openshift image from the
+# openshift registry.
+if [ "$CUSTOM_BASE_IMAGE" == "true" ]; then
```

Comment:
Maybe use non-empty CUSTOM_REPO_FILE as a flag?

Reply:
CUSTOM_REPO_FILE is not required for a custom base image, we may want to rebuild the base image also for other reasons